app/admin/golem/nightshift/page.tsx

```
@@ -1,19 +1,233 @@
+    <div className="flex h-full flex-col overflow-hidden">
+      {/* Header */}
+      <div className="shrink-0 flex items-center justify-between pb-4">
+        <h1 className="text-lg font-semibold text-white flex items-center gap-2">
+          <Moon className="h-5 w-5 text-indigo-400" />
+          Night Shift
+        </h1>
+        <button
+          onClick={refresh}
+          disabled={loading}
+          className="flex items-center gap-2 rounded-lg border border-white/10 px-3 py-2 text-sm text-white/60 hover:bg-white/5"
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)